bpf/bpf_lxc.c

```diff
@@ -1251,13 +1251,9 @@ ipv4_forward_to_destination(struct __ctx_buff *ctx, struct iphdr *ip4,
 
 		ret = fib_redirect_v4(ctx, ETH_HLEN, ip4, false, false, ext_err, &oif);
 		/* Error handling for local routes - just pass the packet to the kernel stack */
-		if (ret == DROP_NO_FIB && *ext_err == BPF_FIB_LKUP_RET_NOT_FWDED) {
-			void *data, *data_end;
-
-			if (!revalidate_data(ctx, &data, &data_end, &ip4))
-				return DROP_INVALID;
+		if (ret == DROP_NO_FIB && *ext_err == BPF_FIB_LKUP_RET_NOT_FWDED)
 			goto pass_to_stack;
-		}
+
```

Comment:
There's `ipv4_l3` once reached `pass_to_stack`, but I don't think we need to revalidate pointers, so LGTM.